pages/wishlist_page/wishlist.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="/assets/s-solid.svg" />
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://kit.fontawesome.com/184c79d39f.js" crossorigin="anonymous"></script>
+    <title> Sierra </title>
+</head>
+<body>
+    <nav class="main-nav">
+        <div class="nav-category">
+            <h3 class="brand-name">S I E R R A</h3>
```

Comment:
Dont manually add the space in html markup. Change it through letter-spacing

```css
landing-spacing: 4px;
```

Reply:
okayyy

Reply:
done